flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/functions/SystemFunctionUtils.java

```diff
@@ -510,4 +510,40 @@ public static Object coalesce(Object... objects) {
         }
         return null;
     }
+
+    public static String convertString(Object object) {
+        return object.toString();
```

Comment:
What would happen if we pass a NULL Object to cast function? Does we allow this happen?